.gitignore

```diff
@@ -14,6 +14,7 @@ coverage
 *.sqlite-*
 apps/*/config/*
 apps/*/*.log
+apps/*/.local
```

Comment:
👍 confirmed that this is also in the .gitignore we generate for client projects

Reply:
Do you think we should put individual `.gitignore` files in these example projects? Or having one at the root that captures what we add to the template we generate on `init` is fine?